SetCurrentVersion.cmd

```diff
@@ -1,7 +1,7 @@
 set MAJOR_PREVIOUS=1
 set MINOR_PREVIOUS=5
 set PATCH_PREVIOUS=12
-set PRERELEASE_PREVIOUS=-beta
+set PRERELEASE_PREVIOUS=-developer
```

Comment:
The _PREVIOUS data is consumed by the helper script that delists the last published version from Nuget. So, on releasing, you update the _PREVIOUS with the last shipped verion. And introduce -developer to the current version to mark it as a working version. Before release, -developer is changed to -beta. Run the script to delist previous. Run the script to publish current. Then repeat as above.